packages/babel-plugin-transform-parameters/src/params.js

```diff
@@ -229,6 +229,6 @@ function buildScopeIIFE(shadowedParams, body) {
   }
 
   return t.returnStatement(
-    t.callExpression(t.arrowFunctionExpression(params, body), params),
+    t.callExpression(t.arrowFunctionExpression(args, body), params),
```

Comment:
@nicolo-ribaudo I believe this is what you intended for. Only one question: do you prefer `args` in `arrowFunctionExpression` or `params` in `arrowFunctionExpression`? 😄

Reply:
According to the spec, a `CallExpression` takes an `ArgumentsList` and an `ArrowFunctionExpression` has some `ArrowParameters`, so it's the other way around :nerd_face: